src/app/auth.service.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/filter';
+import * as auth0 from 'auth0-js';
```

Comment:
Do we need to install auth0-js? This is the file that is causing issues with the image posted above.

Reply:
Yes. `npm install auth0-js` should work

Reply:
I just added it as a dependency so that building it when we deploy goes through.